src/Row.ts

```diff
@@ -37,15 +37,44 @@ class Row extends RowBase<RowProperties> {
 				classes: this.classes(tableCss.table, css.rowTable)
 			}, [
 				v('tr', columns.map((column) => {
-					const { id, field } = column;
+					const { field, id } = column;
+
+					let value: any;
+					if (column.get) {
+						// Get the value from the column callback
+						value = column.get(item, column);
+					}
+					else if (!column.render) {
+						// Get the value using a property lookup on the item data
+						value = item.data[ field || id ];
+					}
+					else {
+						// This column has no value
+						value = '';
+					}
+
+					let content: DNode;
+					if (column.render) {
```

Comment:
I had considered the options API, but with this covering the most common use cases in the first arguments, it doesn't seem worth the overhead.

Do you have any suggestions for a better name?

`render` may be used without `get` and `get` may be used without `render`. Combining everything into a single function will tightly couple the customization our users have traditionally done as well as force users to implement what are pretty standard defaults.